sm-core/src/test/java/com/salesmanager/test/catalog/ProductTest.java

```diff
@@ -513,7 +513,7 @@ private void testAttributes(Product product) throws Exception {
 	    
 	    List<ProductAttribute> attributes = productAttributeService.getProductAttributesByCategoryLineage(store, product.getCategories().iterator().next().getLineage(), en);
 	    
-	    Assert.assertEquals(new Long(attributes.size()).longValue(), 3l);
+	    Assert.assertEquals(new Long(attributes.size()).longValue(), 6l);
```

Comment:
why autoboxing and unboxing!

Reply:
that's there test not mine, it's not related to parts we touched, I found it not working also on there repo, so I fixed it to complete the build